users/models.py

```diff
@@ -0,0 +1,42 @@
+from django.contrib.auth.models import AbstractBaseUser, AbstractUser
+from django.db import models
+
+
+class MyUser(AbstractUser):
+    email = models.EmailField(
+        verbose_name='email address',
+        max_length=255,
+        unique=True,
+    )
+    date_of_birth = models.DateField(blank=True, null=True)
+    is_active = models.BooleanField(default=True)
+    is_staff = models.BooleanField(default=False)
+
+    # objects = MyUserManager()
+
+    USERNAME_FIELD = 'email'
+    REQUIRED_FIELDS = ['username']
+
+    def get_full_name(self):
+        # The user is identified by their email address
```

Comment:
Changed, although 'their' can be used to refer to a person of undetermined gender and is more inclusive than his/her.